exporter/awsemfexporter/datapoint.go

```diff
@@ -122,13 +134,14 @@ func (dps numberDataPointSlice) At(i int) (dataPoint, bool) {
 		metricVal = float64(metric.IntValue())
 	}
 
-	retained := true
+	var retained bool = true
+	var datapoints []dataPoint = []dataPoint{}
```

Comment:
```suggestion
```

Don't think this is needed if the return locations are changed as suggested.

Reply:
Done !! Thanks for your insight !!!